tests/telemetry/conftest.py

```diff
@@ -175,6 +177,9 @@ def test_eventd_healthy(duthosts, enum_rand_one_per_hwsku_hostname, ptfhost, ptf
     if duthost.is_multi_asic:
         pytest.skip("Skip eventd testing on multi-asic")
 
+    reset_event_counters(duthost)
+    restart_eventd(duthost)
```

Comment:
Eventd is already restarted as part of this fixture.

Reply:
I see that your previous commit updated this conftest to include the restart. Current change not needed anymore.